server/clearbit/mapping.js

```diff
@@ -67,5 +67,18 @@ export default function getUserTraitsFromPerson(
     {}
   );
 
+  // Overcome the value:key restrictions and add last_name and first_name to top level traits
```

Comment:
Probably better to use the multiple-destination syntax in the mapper directly no ? : 
https://www.npmjs.com/package/object-mapper#string

Reply:
@michaloo this seems like it could be improved with the above link